internal/manifest/v1beta1/custom_resource.go

```diff
@@ -79,6 +86,23 @@ func PreDeleteDeleteCR(
 		return err
 	}
 
+	var crd unstructured.Unstructured
+	crd.SetName(GetModuleCRDName(obj))
+	crd.SetGroupVersionKind(schema.GroupVersionKind{
+		Version: "v1",
+		Group:   "apiextensions.k8s.io",
+		Kind:    "CustomResourceDefinition",
+	})
+	crdCopy := crd.DeepCopy()
```

Comment:
I don't think you need to `DeepCopy()` - it's not the object read using the client, it was just created in line 89?

Reply:
the reason I did this is that skr here is declarative.Client not Client.client so the skr.Delete needs an *Unstructured

Reply:
:+1